### PR TITLE
Homogenize also rails help command output

### PR DESCRIPTION
**Pull request by albertoalmagro**
### Summary

In commit 6567464bedd1e39ee7390da9484ba0caa7eb3e07 we homogenized rails commands with former rake tasks. We decided to display all commands at the same level and merged the list of commands displayed by `rails help` in the guides.

We however forgot to actually merge the output in the command itself. This patch fixes that.

Thanks in advance for your review!

r? @matthewd

### CODE REVIEW

**Comment by oniofchaos** [comment]
LGTM 👍

**Comment by albertoalmagro** [comment]
Any thoughts @matthewd ? Anyone else? Thanks!

**Comment by albertoalmagro** [comment]
CR please 🙏

File: railties/lib/rails/command.rb [hunk 1]
```diff
@@ -83,20 +83,21 @@ def root
...
 
-      def sorted_groups # :nodoc:
-        lookup!
+      private
+        COMMANDS_IN_USAGE = %w(generate console server test test:system dbconsole new)
+        private_constant :COMMANDS_IN_USAGE
 
-        groups = (subclasses - hidden_commands).group_by { |c| c.namespace.split(":").first }
-        groups.transform_values! { |commands| commands.flat_map(&:printing_commands).sort }
+        def commands # :nodoc:
```
Comment (rafaelfranca) [inline railties/lib/rails/command.rb]:
```suggestion
        def commands
```

**Comment by albertoalmagro** [comment]
@rafaelfranca thanks SO much for the feedback 🙏 Changes applied 😄

**Pull request merged**